third_party/terraform/utils/iam_healthcare_consent_store.go.erb

```diff
@@ -0,0 +1,132 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+
+	healthcare "google.golang.org/api/healthcare/v1beta1"
+
+	"github.com/hashicorp/errwrap"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
+var IamHealthcareConsentStoreSchema = map[string]*schema.Schema{
```

Comment:
We have support for generating IAM resources with Magic Modules. Adding support (and tests, docs) should be as easy as adding an `iam_policy` block onto the resource like the following: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/artifactregistry/api.yaml#L57

You may need more or less of the fields on it to be set depending on how different from the norm this IAM resource is

Looking at how this resource works, it may be similar to this IAM policy: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/datacatalog/api.yaml#L564

Note the `%` character in the base_url and import format. This allows a field to use `/` characters without being escaped, as the `dataset` field will be in the form of `projects/project/...` rather than a raw dataset name

Reply:
Do you mean I can adding the `iam_policy` in api.yaml, then remove `third_party/terraform/utils/iam_healthcare_consent_store.go.erb`?

Reply:
Yes, and you can remove the associated test as tests will be generated automatically.